Automattic/jetpack
# Themes: Hide another "Theme Showcase" broken action

Follow-up of https://github.com/Automattic/jetpack/pull/36986

## Proposed changes:

Removes the broken "Theme Showcase" action from the active theme details that are visible when only one single theme is installed on a site.

Before | After
--- | ---
<img width="1107" alt="Screenshot 2024-05-06 at 17 18 55" src="https://github.com/Automattic/jetpack/assets/1233880/ac6e73f6-c5ec-4edd-8fcc-683e0ad4511e"> | <img width="1307" alt="Screenshot 2024-05-06 at 18 31 40" src="https://github.com/Automattic/jetpack/assets/1233880/6dcc601b-ad67-4be9-8b5c-92fbea720990">



### Other information:


## Jetpack product discussion
N/A

## Does this pull request change what data or activity we track or use?
No

## Testing instructions:
- Apply these changes to your untangled WP.com site
- Go to `wp-admin/themes.php`
- Delete all the themes except the active one (you need to have only one single theme installed to reproduce the original issue)
- The theme details should became automatically visible
- Make sure that the theme actions at the bottom don't include a "Theme Showcase" button
